src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -96,6 +96,10 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
     @computed get molecularProfileIdToMolecularProfile() {
         return _.keyBy(this.profiles, profile=>profile.molecularProfileId);
     }
+    
+    private sortCoExpressionData(data: any[]): any[] {
+        return _.sortBy(data, ["pValue", "hugoGeneSymbol"]);
```

Comment:
another way to do this, which may be a little cleaner, is to adjust the `sortBy` function for the qValue column to be d=>[d.qValue, d.pValue]